core/editor.ts

```diff
@@ -62,8 +63,10 @@ class Editor {
             ) {
               isImplicitNewlineAppended = true;
             }
-            const [leaf] = this.scroll.leaf(index);
-            const formats = merge({}, bubbleFormats(leaf));
+            // @ts-expect-error
+            const [leaf] = this.scroll.descendant(LeafBlot, index);
+            // @ts-expect-error
+            const formats = merge({}, bubbleInlineParentFormats(leaf));
```

Comment:
When `leaf` is an inline embed, `bubbleFormats(leaf)` also includes the formats of the inline embed. However, the inserted inline embed won't inherit those formats. Consequently, when an inline embed is inserted before another one with the same formats, the newly inserted inline embed will not have any formats applied to it.